src/reducers/profile-view.js

```diff
@@ -894,3 +895,99 @@ export const selectedThreadSelectors: SelectorsForThread = (() => {
   const result2: SelectorsForThread = result;
   return result2;
 })();
+
+export type SelectorsForNode = {
+  getName: State => string,
+  getIsJS: State => boolean,
+  getLib: State => string,
+  getTimingsForSidebar: State => TimingsForPath,
+};
+
+export const selectedNodeSelectors: SelectorsForNode = (() => {
+  const getName = createSelector(
```

Comment:
I think `getLeafFunctionName` would be more descriptive here.

Reply:
I'm not sure I agree here. This _is_ the name of the selected node we're getting here.